plugins/modules/elastic_endpoint_security.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/python
+# Copyright 2021 Expedient
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ansible.module_utils.basic import _ANSIBLE_ARGS, AnsibleModule
+
+try:
+  from ansible_collections.expedient.elastic.plugins.module_utils.kibana import Kibana
+except:
+  import sys
+  import os
+  util_path = new_path = f'{os.getcwd()}/plugins/module_utils'
+  sys.path.append(util_path)
+  from kibana import Kibana
+
+results = {}
+
+class SecurityBaseline(Kibana):
+    def __init__(self,module):
+        super().__init__(module)
+        self.module = module
+        self.integration_pkg_name = module.params.get('integration_pkg_name')
+        self.integration_pkg_desc = module.params.get('integration_pkg_desc')
+        self.endpoint_security_antivirus = module.params.get('endpoint_security_antivirus')
+        self.prebuilt_rules_activate = module.params.get('prebuilt_rules_activate')
+        self.agent_policy_name = self.module.params.get('agent_policy_name')
+        self.agent_policy_desc = self.module.params.get('agent_policy_desc')
+        self.pkg_policy_name = self.module.params.get('pkg_policy_name')
+        self.pkg_policy_desc = self.module.params.get('pkg_policy_desc')
+        
+    def create_securityctrl_baseline_settings(self,pkg_policy_object):
+        ################ Checking and creating package policy associated with Integration
+        kibana = Kibana(self.module)
```

Comment:
`Kibana` already exists here as `self` so all instances of `kibana` can be replaced with `self` inside this function.